plugin_runner/plugin_runner.py

```diff
@@ -294,17 +307,22 @@ async def server_graceful_shutdown() -> None:
     await server.wait_for_termination()
 
 
-if __name__ == "__main__":
+def run_server(specified_plugin_paths: list[str] | None = None) -> None:
```

Comment:
This is imported and invoked by the CLI, as well as being called by the `if __name__ == "main":` below.